Bulk Import help page updates

goddesswarship opened the pull request:
Updating help page to account for workflow changes in Wildbook 10.8.0

goddesswarship committed:
Bulk Import help page updates

updating help page to account for workflow changes in Wildbook 10.8.0

goddesswarship committed:
Update bulk-import-beta.md

Updates to workflow steps made. Will continue to edit as QA is updated.

goddesswarship committed:
Update bulk-import-beta.md

goddesswarship committed:
Update bulk-import-beta.md

goddesswarship committed:
Update bulk-import-beta.md

goddesswarship committed:
Update bulk-import-faq.md

Removed question relating to mismtached name and id fields in locationID.json. This will be resolved by https://github.com/WildMeOrg/Wildbook/issues/711

goddesswarship committed:
Update bulk-import-beta.md

goddesswarship committed:
Update bulk-import-beta.md

goddesswarship committed:
Update bulk-import-beta.md

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -1,49 +1,41 @@
-* Photo archive in a local file system
-* Excel spreadsheet linking photos to metadata
+* A folder that contains only your photos (no videos)
+* An Excel spreadsheet with the appropriate fields and metadata entered
```
or csv

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -1,49 +1,41 @@
+## Spreadsheet
 
-In the header of an Excel (.xlsx) file, create a column for each field you want to upload to Wildbook. [See Fields Available for a list of supported fields.](#fields-available) This must include at least:
+In an Excel (.xlsx) file, create a column for each field you want to upload to Wildbook. [See Fields Available for a list of supported fields and their descriptions.](#fields-available) This must include:
```
or csv file

vkirkl commented on docs/data/bulk-import-beta.md:
````diff
@@ -1,49 +1,41 @@
-
-## Photo Archive Set-up
+```{note}
+Your spreadsheet should have 200 Encounters or less. Spreadsheets larger than this will slow down the site for all users and will result in longer wait times for detection and identification.
````
"... or fewer." rather than "... or less."

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -55,40 +47,30 @@ Here are example Bulk Import Excel spreadsheet templates:
+1. Click **Browse** to select your images or drag and drop them into the box.
+2. Once each photo upload reaches 100%, click **Next**.
+3. Click **Browse** to select your spreadsheet or drag and drop it into the box.
+4. Once spreadsheet upload reaches 100%, click **Next**.
```
note that we have played with auto-advancing to data review since the spreadsheet upload is so quick ... double check that this step is still necessary.

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -55,40 +47,30 @@ Here are example Bulk Import Excel spreadsheet templates:
-6. Select **Begin Upload** to be taken to the import overview.
-7. Review the data preview for data integrity.
+1. Click **Browse** to select your images or drag and drop them into the box.
+2. Once each photo upload reaches 100%, click **Next**.
```
note:  the user can advance before all image uploads complete ... but it probably a good doc practice to suggest that they wait.

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -55,40 +47,30 @@ Here are example Bulk Import Excel spreadsheet templates:
+2. Once each photo upload reaches 100%, click **Next**.
+3. Click **Browse** to select your spreadsheet or drag and drop it into the box.
+4. Once spreadsheet upload reaches 100%, click **Next**.
+5. Review your spreadsheet for errors. Errors, missing data, or empty fields will be highlighted for you to correct.
```
How about a minor change:  "Review the data uploaded from your spreadsheet and correct any validation errors and missing data.  Errors, missing data, or empty fields will be highlighted."

Should we note the "set all values in this column" functionality for location id and submitter id?

vkirkl reviewed (commented):
Looking good!  
A few suggestions inline.

goddesswarship committed:
Update bulk-import-beta.md

Updated delete import instructions

goddesswarship committed:
Update bulk-import-beta.md

Incorporated @vkirkl's feedback

goddesswarship committed:
Update bulk-import-beta.md

Updated Bulk Import Matching Process section with more details about viewing matches from bulk import page.

vkirkl commented on docs/data/bulk-import-beta.md:
```diff
@@ -64,7 +64,20 @@ If any of the steps above are interrupted due to loss of internet connection or
+
+As identification progresses, the [IA Class](../data/manual-annotation-beta.md#creating-a-manual-annotation) will appear in the **Class** column of the table along with the match results status:
+
+* waiting to queue - Images haven't sent to [WBIA](../introduction/index.md) yet
```
How about "Images haven't been sent to ..." 
Super picky!  :) 

Also, some of these have periods at the end and some don't.  Let's make them all one way or the other.

goddesswarship committed:
Update bulk-import-beta.md

Missed a word in line 71

goddesswarship committed:
Update bulk-import-beta.md

punctuation corrections in likes 71-78

vkirkl reviewed (approved):
Woohoo!
